modules/product/src/main/java/com/opengamma/strata/product/index/type/ImmutableOvernightFutureContractSpec.java

```diff
@@ -70,6 +72,13 @@ public final class ImmutableOvernightFutureContractSpec
    */
   @PropertyDefinition(validate = "notNull")
   private final DateSequence dateSequence;
+  /**
+   * The accrual period of the future.
+   * <p>
+   * This is the period of time over which the daily interest rate fixing is observed.
+   */
+  @PropertyDefinition(get = "optional")
```

Comment:
Would prefer to make this non-optional - but would make this class non backwards compatible. It's used in a couple of internal repos, but we could easily fix